### PR TITLE
Bozja Lyon module tentatively finished

### PR DESCRIPTION
**Pull request by CarnifexOptimus**
Only tested on replays, hope someone can test it in the live game.

### CODE REVIEW

File: BossMod/Modules/Shadowbringers/Foray/Duel/Duel2Lyon/Duel2Lyon.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
+namespace BossMod.Shadowbringers.Foray.Duel.Duel2Lyon
+{
+
+class Lyon2DuelStates : StateMachineBuilder
```
Comment (awgil) [inline BossMod/Modules/Shadowbringers/Foray/Duel/Duel2Lyon/Duel2Lyon.cs:4]: Looks like something is off with your formatting...

File: BossMod/Network/PacketInterceptor.cs [hunk 2]
```diff
@@ -42,8 +42,10 @@ public bool Active
 
         public unsafe PacketInterceptor()
         {
-            // alternative signature: E8 ?? ?? ?? ?? 84 C0 0F 85 ?? ?? ?? ?? 48 8D 35 - seem to be changing from build to build
-            _fetchHook = Service.Hook.HookFromSignature<FetchReceivedPacketDelegate>("E8 ?? ?? ?? ?? 84 C0 0F 85 ?? ?? ?? ?? 44 0F B6 64 24", FetchReceivedPacketDetour);
+            // alternative signatures - seem to be changing from build to build:
+            // - E8 ?? ?? ?? ?? 84 C0 0F 85 ?? ?? ?? ?? 48 8D 35
+            // - E8 ?? ?? ?? ?? 84 C0 0F 85 ?? ?? ?? ?? 44 0F B6 64 24
+            _fetchHook = Service.Hook.HookFromSignature<FetchReceivedPacketDelegate>("E8 ?? ?? ?? ?? 84 C0 0F 85 ?? ?? ?? ?? 48 8D 35", FetchReceivedPacketDetour);
```
Comment (awgil) [inline BossMod/Network/PacketInterceptor.cs:48]: Mismerge?

File: BossMod/Modules/Shadowbringers/Foray/Duel/Duel2Lyon/Duel2Lyon.cs (hunk 1 above)
Comment (CarnifexOptimus) [reply BossMod/Modules/Shadowbringers/Foray/Duel/Duel2Lyon/Duel2Lyon.cs]: ok I changed the formating

File: BossMod/Network/PacketInterceptor.cs (hunk 2 above)
Comment (CarnifexOptimus) [reply BossMod/Network/PacketInterceptor.cs:48]: Is it a problem? don't think it will change anything considering it is identical with the existing code

**Pull request merged**